src/churn/churn.ts

```diff
@@ -271,9 +273,34 @@ export var filterCandidatesFromSdp = (sdp:string) : string => {
     private configurePipe_ = (key:number) : void => {
       this.pipe_ = freedom['churnPipe'](this.peerName);
       this.pipe_.on('mappingAdded', this.onMappingAdded_);
-      this.pipe_.setTransformer('caesar',
+      /*this.pipe_.setTransformer('none',
```

Comment:
This is all a bit messy.  There seems to be quite a bit of magic going on in here.